README.md

```diff
@@ -67,28 +67,31 @@ make m3
 - Run `m3 setup` on the local kubernetes
 
 ```
-./m3 setup "Admin Name" admin@email.com
+./m3 setup
 ```
 
-This will setup `m3` and create the first admin account **Note down the admin access/secret key**
+- Make postgres reachable from host OS
 
-You may see the following error message
 ```
-Running Migrations
-2019/10/17 12:02:50 error connecting to database or reading migrations
-2019/10/17 12:02:50 dial tcp 127.0.0.1:5432: connect: connection refused
```

Comment:
Since `SetupM3` function is still doing `setupPostgres()` this  message should still be after `./m3 setup` since otherwise you won't know if you need to do the port-forward or not.

Reply:
oh forget about what I said, I realized `RunMigrations ` was removed from `./m3 setup`